client/src/App.vue

```diff
@@ -15,7 +15,7 @@
     v-toolbar.app-toolbar(app, fixed, clipped-left)
       v-toolbar-side-icon(@click.stop='drawer = !drawer', v-if='user.isAdmin')
       v-toolbar-title.mr-3
-        .headline.cursor-pointer(@click='$router.push("/")') things
+        .headline.cursor-pointer(@click='$router.push("/")') Home
```

Comment:
Can we make this redirect to /home if the user has logged in? If I'm not mistaken this will take you back to the login page.

Reply:
That's true. `App.vue` has a `user` data property we can use to check. We should probably change the `user` property in `data()` from `{id:0}` to `{}` first